Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:wheezy
+MAINTAINER Di Wu <diwu@yelp.com>
+
+COPY . /opt/redash/current/
+
+# Install dependencies
+WORKDIR /opt/redash/current
+RUN bash setup/bootstrap_docker.sh
+
+# Build frontend sources
+WORKDIR /opt/redash/current/rd_ui
+USER redash
+RUN mkdir -p /opt/redash/.npm/node_modules
+RUN npm config set prefix /opt/redash/.npm/node_modules
+RUN npm install -g bower
+RUN npm install -g grunt-cli
+RUN npm install
+RUN /opt/redash/.npm/node_modules/bin/bower install
+RUN /opt/redash/.npm/node_modules/bin/grunt build
+
+# Reset working directory and user for future `docker attach` sessions
+WORKDIR /opt/redash/current
+USER root
+
+CMD service redash_supervisord start
```

Comment:
Missed that, but anyway no need to run it as service/with init. Just run
the supervisor command.
On Oct 6, 2015 9:27 PM, "John Wu" <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/EverythingMe/redash/pull/588#discussion_r41302816>:
>
> > +# Build frontend sources
> > +WORKDIR /opt/redash/current/rd_ui
> > +USER redash
> > +RUN mkdir -p /opt/redash/.npm/node_modules
> > +RUN npm config set prefix /opt/redash/.npm/node_modules
> > +RUN npm install -g bower
> > +RUN npm install -g grunt-cli
> > +RUN npm install
> > +RUN /opt/redash/.npm/node_modules/bin/bower install
> > +RUN /opt/redash/.npm/node_modules/bin/grunt build
> > +
> > +# Reset working directory and user for future `docker attach` sessions
> > +WORKDIR /opt/redash/current
> > +USER root
> > +
> > +CMD service redash_supervisord start
>
> Yeah, that's what I did to supervisord_docker.conf by setting
> nodeamon=true
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EverythingMe/redash/pull/588/files#r41302816>.
>